### DISCUSSION TITLE
LTSP with Remmina Kiosk

### DISCUSSION THREAD

**Discussion by hobelchen** [ltsp/ltsp#689] [Q&A]
I did the the  steps from the wiki page https://github.com/ltsp/ltsp/wiki/Remmina-Kiosk-(RDP,-Spice,-&-VNC) and always failed to get a connection to my xrdp server and the system crashed. 
As far as I found out remmina wasnt able to write into the location  /var/run/home/kiosk/.local/share/remmina/

`** (org.remmina.Remmina:2162): WARNING **: 11:49:45.477: (remmina_file_save) - Remmina connection profile cannot be saved, with error 2 (Failed to create file “/run/home/kiosk/.local/share/remmina/rdp.remmina.NK3AJ1”: Keine Berechtigung)`

After I changed the corresponding line of the code to: chown kiosk: /var/run/home/kiosk/.local/share/remmina without the /* in the end it worked for me. 

And the second step of the wiki is not necessary. I installed "remmina-gnome" and it worked fine. (ppa:remmina-ppa-team/remmina-next-daily) 

Shall I chance the wiki? Can anyone confirm that this is right?

**Comment by alkisg** [comment]
I converted this issue into a discussion as it's related to the community wiki, not to the upstream LTSP code base.
You might also check who wrote that wiki page and @ping them in case they're interested in helping you.
